catalog/tests/dags/providers/provider_api_scripts/test_science_museum.py

```diff
@@ -166,7 +166,7 @@ def test_get_record_data_success(object_data):
     expected_image_data = {
         "foreign_identifier": "i458349",
         "foreign_landing_url": "https://collection.sciencemuseumgroup.org.uk/objects/co84905/small-votive-organ-of-unknown-type-votive-viscera",
-        "url": "https://coimages.sciencemuseumgroup.org.uk/458/349/large_a659676__0001_.jpg",
+        "url": "https://coimages.sciencemuseumgroup.org.uk/458/349/large_a659676__0001_.jpg/",
```

Comment:
This adding of a trailing slash is a little concerning - I tried to access the new URL and with the trailing slash I got a 403, but without the slash I was able to view the image. We may need to adjust the logic (both in the ingestion DAG and the cleanup one) to account for this.

Reply:
It does not add a slash; this is done by our mocks for the [tests](https://github.com/WordPress/openverse/blob/main/catalog/tests/dags/conftest.py#L146) but does not appear in our data. Good eye, though :) I can add a comment at least or mock the entire `rewrite_redirected_url`.

Reply:
Whew, that's a relief!